### DISCUSSION TITLE
Linear Blend Skinning vs. Compute Deform

### DISCUSSION THREAD

**Discussion by sdrowan123** [Dreaming381/Latios-Framework#13] [Q&A] [answered]
How is the performance between these two methods? Would anyone be able to share some pros and cons from their experience?

**Comment by Dreaming381** [comment, marked as answer]
LBS is usually 50-100% faster in my tests on my old R9 390 (which is a very compute oriented card where I would have expected Compute Deform to win). LBS requires significantly less memory bandwidth and tends to schedule more optimally.

Use Compute Deform if you need more than 4 bone weights or in a future release if you need Blend Shapes.

With that said, Kinemation's Compute Deform is 50% faster than the Hybrid Renderer's Compute Deform implementation in my testing on my GPU, and that was with culling disabled in Kinemation. So if anyone is using raw Hybrid Renderer for comparison, that needs to be taken into account.
